alexa-news.php

```diff
@@ -32,7 +32,47 @@ public function news_request( $event ) {
 
 		switch ( $intent ) {
 			case 'Latest':
-				$result = $this->endpoint_content();
+				$term_slot = strtolower( sanitize_text_field( $request->getSlot( 'TermName' ) ) );
+
+				$args = [
+					'post_type'      => alexawp_news_post_types(),
+					'posts_per_page' => max( 1, min( 100, count( $this->placement ) ) ),
```

Comment:
ok, I was thinking about it from a UX side, having 100 headlines read to ya seems unlikely before one falls asleep :)